pandas/core/sparse/dtype.py

```diff
@@ -173,9 +173,10 @@ def construct_from_string(cls, string):
             'Sparse[int, 1]' SparseDtype[np.int64, 0]
```

Comment:
I think this one now needs to be updated as the above would raise an error. But make it `'Sparse[int, 0]` to show that default fill value is OK?